### PR TITLE
Condarc #29

### PR DESCRIPTION
**Pull request by cmeesters**
made the inclusion of a tex file with a sample condarc configurable

### CODE REVIEW

**Comment by mptrsen** [comment]
How about:

1. Have a regular `condarc` (not a complete slide), i.e. just a copy of the actual, site-specific `.condarc`
2. Put the path to it (can be `setup/condarc`) into the config
3. Keep a generic `slides/common/condarc.tex` and use `\lstinputlisting[language=Bash, style=Shell, basicstyle=\tiny]{\configparam{condarcfile}}`

That would eliminate the need to copy+paste the *content* of `.condarc` into the slide **and** have site-specific .tex files

**Comment by mptrsen** [comment]
>     3. Keep a generic slide and use `\lstinputlisting[language=Bash, style=Shell, basicstyle=\tiny]{\configparam{condarcfile}}`

Edited to say "keep generic slide"

**Comment by cmeesters** [comment]
Sounds enticing. Alas, listings are verbatim (no interpreted commands in verbatim). Plus, the way it is, people can have their own slide content (e.g. different remarks).

**Comment by mptrsen** [comment]
That's true. There will be no interpreted commands in the condarc to be included, but the flexibility in the slide content is worth a bunch.

File: slides/Snakemake_HPC_Creators.tex [hunk 1]
```diff
@@ -7,7 +7,10 @@
 % to typeset only a few slide sets, set them here during development
 %\includeonly{Why_Workflows}
 
-\include{common/preamble}
+
+\input{common/preamble}
```
Comment (mptrsen) [inline slides/Snakemake_HPC_Creators.tex:11]: good catch!

**Review by mptrsen — APPROVED** [review]
seems good

**Pull request merged**